docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Can you please match the plotting style from here:</p><p> </p><p>https://stumpy.readthedocs.io/en/latest/Tutorial_STUMPY_Basics.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>About second plot, if these data is plotted with original plotting style, These data does not look like "<strong>not well conserved" by streching too widely.</strong></p><p><strong>So I limited the width.</strong></p>

Reply:
Now I undarstand. All plot should be "stumpy style". I bascically reproduced paper style.